templates/bot/ts/command-and-response/README.md

```diff
@@ -11,7 +11,7 @@ The app template is built using the TeamsFx SDK, which provides a simple set of
 >
 > To run the command bot template in your local dev machine, you will need:
 >
-> - `Node.js` installed locally (recommended version: 14)
+> - `Node.js` installed locally (recommended version: 16)
```

Comment:
If change the recommended version, also need to update the `notification-restify` readme.